src/snowflake/snowpark/session.py

```diff
@@ -982,6 +983,9 @@ def write_pandas(
                 table, you can always create your own table before calling this function. For example, auto-created
                 tables will store :class:`list`, :class:`tuple` and :class:`dict` as strings in a VARCHAR column.
             create_temp_table: The to-be-created table will be temporary if this is set to ``True``.
+            overwrite: When true, and if auto_create_table is true, then it drops the table. Otherwise, it
+                truncates the table. In both cases it will replace the existing contents of the table with that of the passed in
+                Pandas DataFrame.
```

Comment:
what would happen if overwrite is the default value `False`?

Reply:
@sfc-gh-aling Let me know if this description is better "_Default value is ``False`` and the Pandas DataFrame data is appended to the existing table. If set to ``True`` and if auto_create_table is also set to ``True``, then it drops the table. If set to ``True`` and if auto_create_table is set to ``False``, then it trunctates the table. Note that in both cases (when overwrite is set to ``True``) it will replace the existing contents of the table with that of the passed in Pandas DataFrame._"

Reply:
Submitting a new PR with the above description.